index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kick UUIDs</title>
+    <script src="https://cdn.jsdelivr.net/npm/protobufjs@7.2.5/dist/protobuf.min.js"></script>
+</head>
+<body>
+    <h1>UUID as a Service</h1>
+    <p>No warranty, nothing. Use at your own risk. Good luck! :-)</p>
+    <input type="text" id="ipInput" placeholder="Enter IP">
+    <input type="text" id="pinInput" placeholder="Pin">
+    <input type="text" id="uuidInput" placeholder="UUID">
+    <button id="connectButton">Kick!</button>
+    <p id="result"></p>
+
+    <script>
+        protobuf.load("https://raw.githubusercontent.com/dupondje/nrgkick-proto/main/nrgcp.proto", function(err, root) {
+            if (err) {
+                throw err;
+            }
+
+            window.Nrgcp = root.lookupType("nrgkick.Nrgcp");
+            window.Nrgcp_Header = root.lookupType("nrgkick.Nrgcp.Header");
+            window.Nrgcp_Metadata = root.lookupType("nrgkick.Nrgcp.Metadata");
+            window.Nrgcp_Payload = root.lookupType("nrgkick.Nrgcp.Payload");
+            window.Nrgcp_Payload_DeviceControl_Info_Update = root.lookupType("nrgkick.NrgcpDevicecontrolInfoUpdatePayload");
+            window.Nrgcp_Payload_DeviceControl_Info_Update_DevicePin = root.lookupType("nrgkick.NrgcpDevicecontrolInfoUpdatePayload.DevicePin");
+            window.Nrgcp_Types = root.lookupType("nrgkick.NrgcpTypes");
+
+        });
+
+        function blobToUint8Array(blob, callback) {
+            const reader = new FileReader();
+            reader.onload = function() {
+                const arrayBuffer = reader.result;
+                const uint8Array = new Uint8Array(arrayBuffer);
+                callback(uint8Array);
+            };
+            reader.onerror = function(error) {
+                console.error("Error reading blob as array buffer:", error);
+            };
+            reader.readAsArrayBuffer(blob);
+        }
+
+        document.getElementById('pinInput').value = '7311122';
+        document.getElementById('uuidInput').value = '86bdf0aa-0468-4689-bd7c-f953cae1b356';
```

Comment:
This is your pin and uuid? Or some example?

Reply:
Ups, I copied this file from another repository where these two settings have not been added/committed. I'll amend this commit/PR. Please delete this comment afterwards so other people can't see it.